tensorflow/python/ops/nn_ops.py

```diff
@@ -945,6 +945,61 @@ def _FusedResizeAndPadConv2DShape(op):
   return [tensor_shape.TensorShape(output_shape)]
 
 
+@ops.RegisterShape("FusedPadConv2D")
+def _FusedPadConv2DShape(op):
```

Comment:
can this use the same function as the other fused one?  This will make it easier when porting to C++ (which will happen soon...)